blocks/property-listing/cards/cards.js

```diff
@@ -38,9 +38,13 @@ export function createCard(listing) {
     item.classList.add('is-featured');
   }
 
+  if (listing.mlsStatus) {
+    item.classList.add('is-new');
```

Comment:
The existence of the MLS Status doesn't indicate that something is new, it has a specific value ('NEW LISTING' i believe)

Reply:
fixed